logid/logid_test.go

```diff
@@ -0,0 +1,134 @@
+// Copyright 2015 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package logid
+
+import (
+	"crypto/sha256"
+	"encoding/base64"
+	"log"
+	"reflect"
+	"testing"
+)
+
+var (
+	wanted LogID = [32]byte{
+		'c', 't', 'l', 'o', 'g', '_', 'i', 'd', '_', 'f', 'o', 'r', '_', 't', 'e', 's',
+		't', 'i', 'n', 'g', '_', 'p', 'u', 'r', 'p', 'o', 's', 'e', 's', '!', '!', '1',
+	}
+)
```

Comment:
See if you can have instead:

wanted LogID = [32]byte("ctlog_id_for_testing_purposes!!1")

Reply:
Gave it a try, but it doesn't convert. :disappointed:

Reply:
Drive-by comment:

Apparently, casting string to byte array doesn't work (only to []byte). You could get fancy with copy, eg:

```go
var logID [32]byte

// Possibly on "init()"
copy(logID[:], []byte("ctlog_id_for_testing_purposes!!1"))
```

Reply:
Seems like more fuss than leaving it as separate bytes, for a test?

Reply:
I mostly got curious about the cast-string-to-array thing. I think it's fine for the test to go either way.

Reply:
I think it's fine to leave as is.

Reply:
I think for a test it's fine to leave this as-is.